.github/FUNDING.yml

```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: # [suchov] Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```

Comment:
added my surname just to make the PR valid, we can think about the options that we want to put